net/chrony/Makefile

```diff
@@ -52,7 +63,7 @@ CONFIGURE_ARGS+= \
 	--sysconfdir=/etc/chrony \
 	--prefix=/usr \
 	--chronyrundir=/var/run/chrony \
-	--disable-nts \
+	$(if $(CONFIG_CHRONY_NTS),,--disable-nts) \
```

Comment:
--$(if $(CONFIG_CHRONY_NTS),en,dis)able-nts \ is better I think.

Reply:
In this case it's not an autoconf-generated configure script and it doesn't have an `--enable-nts` option. The detection of gnutls and nettle, which are required for NTS, can be just disabled.

Reply:
OK.